lock-api/src/main/java/com/palantir/lock/client/LeasedLockToken.java

```diff
@@ -83,4 +84,13 @@ public UUID getRequestId() {
     public SafeArg<?> toSafeArg(String name) {
         return SafeArg.of(name, serverToken);
     }
+
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper("LeasedLockToken")
+                .omitNullValues()
+                .add("serverToken", serverToken)
+                .add("requestId", requestId)
```

Comment:
ack. The Lease I think just has a bunch of leader time internals... it might be useful, but yeah not sure we should pay the synchronization cost for that